packages/cli/src/commands/database/ogcio/ogcio-seeder-local.json

```
@@ -807,8 +807,8 @@
         "name": "FormsIE Admin",
         "description": "The form manager admin App",
         "type": "SPA",
-        "redirect_uri": "http://localhost:4000/admin/auth/callback",
-        "logout_redirect_uri": "http://localhost:4000",
+        "redirect_uri": "http://localhost:4000/admin/v2/auth/callback",
+        "logout_redirect_uri": "http://localhost:4000/admin/v2/overview/",
```

```suggestion
        "redirect_uri": [
            "http://localhost:4000/admin/v2/auth/callback",
            "http://localhost:4000/admin/auth/callback"
        ],
        "logout_redirect_uri": [
            "http://localhost:4000/admin/v2/overview/",
            "http://localhost:4000"
         ],
```

To keep compatibility with the formsie logto playground